packages/cardhost/app/components/card-container/index.hbs

```diff
@@ -1,16 +1,19 @@
-<Boxel::CardContainer @displayBoundaries={{true}} style="margin:1vw 2vw;overflow:hidden;">
+<Boxel::CardContainer class="card-container" @displayBoundaries={{true}} ...attributes>
   <Boxel::Header @header="Layout">
```

Comment:
Added Boxel::Header component here because the edit buttons that need to be on the header are here. According to this, the isolated card container will always be called "Layout", but that may not always be the case.